storage/mysql/log_storage.go

```diff
@@ -567,11 +567,26 @@ func (t *logTreeTX) GetLeavesByIndex(ctx context.Context, leaves []int64) ([]*tr
 
 func (t *logTreeTX) GetLeavesByRange(ctx context.Context, start, count int64) ([]*trillian.LogLeaf, error) {
 	if count <= 0 {
-		return nil, fmt.Errorf("invalid count %d", count)
+		return nil, fmt.Errorf("count=%d,want>0", count)
```

Comment:
Not sure about changing these. I think they might get returned in server responses but not 100% certain.

Reply:
Yes, it gets returned in servers responses. Should I leave it intact? I thought a bit clarified messages would be better.

Reply:
Made it slightly more human readable.